qemu/tests/cfg/passt_transfer_tests.cfg

```diff
@@ -0,0 +1,31 @@
+- passt_transfer_tests:
+    virt_test_type = qemu
+    type = passt_transfer_tests
+    image_snapshot = yes
+    vhost_nic1 = ""
+    nettype_nic1 = user:passt
+    net_port_forwards_avocado-vt-vm1 = TCP@10001 UDP@10001
+    net_port_forwards_vm2 = TCP@10002 UDP@10002
+    vms = "avocado-vt-vm1 vm2"
+    tmp_dir = "/var/tmp/"
+    fw_stop_cmd = systemctl stop firewalld || nft flush ruleset || iptables -F
+    file_md5_check_timeout = 120
+    filesize = 4294967296
+    dd_cmd = "dd if=/dev/zero of=%s bs=%d count=1"
+    variants:
+        - with_tcp_ipv4:
+            receive_cmd = socat -u TCP-LISTEN:%d,reuseaddr OPEN:%s,create
+            sent_cmd = socat -u FILE:%s TCP:%s:%d
+        - with_udp_ipv4:
+            filesize = 51200
+            receive_cmd = socat -u UDP4-LISTEN:%d,reuseaddr OPEN:%s,create & echo $! >/tmp/socat.pid && sleep 60 && kill -9 $(cat /tmp/socat.pid)
+            sent_cmd =  socat -u FILE:%s UDP4:%s:%d
+        - with_tcp_ipv6:
+            ipv6 = yes
+            receive_cmd = socat -u TCP6-LISTEN:%d,reuseaddr OPEN:%s,create
+            sent_cmd = socat -u FILE:%s TCP6:[%s]:%d
+        - with_udp_ipv6:
+            ipv6 = yes
+            filesize = 51200
+            receive_cmd = socat -u UDP6-LISTEN:%d,reuseaddr OPEN:%s,create & echo $! >/tmp/socat.pid && sleep 60 && kill -9 $(cat /tmp/socat.pid)
+            sent_cmd = socat -u FILE:%s UDP6:[%s]:%d
```

Comment:
Thanks for your suggestion, It will  be udpate next time.

Reply:
update, after testing, I have to change this line `sent_cmd = socat -u FILE:%s ${receive_protocol}:[%s]:%d`, since it can not test pass on ipv4 address due to `[%s]` format, I will try to find a available method.